packages/cactus-plugin-bungee-hermes/package.json

```diff
@@ -63,6 +63,7 @@
     "@hyperledger/cactus-plugin-ledger-connector-fabric": "2.0.0-alpha.2",
     "axios": "1.6.0",
     "body-parser": "1.20.2",
+    "crypto-js": "4.2.0",
```

Comment:
if possible to use crypto functions from Cacti common use them instead of adding a new dep.

Reply:
I think there is no utility function to hash in cactus common. I just use the module to hash the privacy policy.

I can change the dependency/function to the default one use in cactus-common perhaps? like 'sha3' module

Reply:
@RafaelAPB @petermetz I changed to a hashData function from private to public in the JsObjectSigner object, in the cactus-common package, so that i can use that instead of the extra dependency

Reply:
@eduv09 @RafaelAPB Works for me, thank you!